src/store/state/state-model.ts

```diff
@@ -193,6 +193,16 @@ export const stateModel: StateModelIF = {
     businessTypeConfirm: false,
     isAutoPopulatedBusinessNumber: false
   },
+  // *** TODO: update
```

Comment:
To be updated by future tickets.

Reply:
I'd like to see this a little more descriptive. Update what?

Reply:
I moved the comment inside the object to be updated and changed it to:
`// *** TODO: add/remove properties as needed`